docs/playground/searching.md

```diff
@@ -0,0 +1,10 @@
+# Text & Image Searching
+
+<iframe frameborder="0" allowtransparency="true" scrolling="no" src="https://jemmyshin-laion5b-streamlit-streamlit-demo-r1byr6.streamlitapp.com/" style="overflow:hidden;overflow-x:hidden;overflow-y:hidden;height:100vh;width:100%"></iframe>
+
+```{button-link} https://jemmyshin-laion5b-streamlit-streamlit-demo-r1byr6.streamlitapp.com/
```

Comment:
https://jemmyshin-laion5b-streamlit-streamlit-demo-rddbqz.streamlitapp.com/

Reply:
why do we need to update the server host? Can we avoid such broken change?